meson.build

```diff
@@ -202,6 +204,13 @@ if host_machine.system() == 'linux'
   conf_data.set('C_ALSA', 1)
 endif
 
+# Windows-only dependencies
+#
+if host_machine.system() == 'windows' or host_machine.system() == 'cygwin'
```

Comment:
``` python
if host_machine.system() in ('windows', 'cygwin')
```
Although maybe not available in meson's DSL?

Reply:
Right now this would work, but it's ugly:
```meson
if ['windows', 'cygwin'].contains(host_machine.system())
```
Starting with meson 0.49.0:
```meson
if host_machine.system() in ['windows', 'cygwin']
```
Soo… as soon as we'll be able to move on to using meson >= 0.49.0, I will replace it (right now we're compatible with meson >= 0.45.1).